calico/security/encrypt-cluster-pod-traffic.md

```diff
@@ -22,17 +28,21 @@ This how-to guide uses the following {{site.prodname}} features:
 
 ### Before you begin...
 
-The following platforms:
+**Unsupported**
+
+- GKE
+- Using your own custom keys to encrypt traffic
+
+**Limitations**
 
-- Kubernetes, on-premises
-- EKS using Calico CNI
-- EKS using AWS CNI
-- AKS using Azure CNI
+- IPv4 only
+- EKS, only with AWS CNI
+- AKS, only with Azure CNI
 
 **Supported encryption**
 
-- Host-to-host encryption for pod traffic
-- Encryption for direct node-to-node communication - supported only on managed clusters deployed on EKS and AKS
+- Pod-to-pod traffic
+- Encryption for direct node-to-node communication is supported only on managed clusters deployed on EKS (AWS CNI) and AKS (Azure CNI)
```

Comment:
We'd need to update this line too if Calico CNI is to be supported for EKS

Reply:
According to Sujeet, we do not support EKS with Calico CNI.

Reply:
> According to Sujeet, we do not support EKS with Calico CNI.
There's confusion here, I think.  I think there are 2 features here.
- One encrypts ALL host-host traffic (including pod traffic) and is only available on EKS and AKS with their respective CNIs.
- The other is where we encrypt only pod-pod traffic between hosts, and that's available with Calico CNI on all supported platforms where we can run wireguard.

Reply:
Based on a very recent round of testing (thanks @aaaaaaaalex ) I think Wireguard can be considered as supported for EKS + CalicoCNI.

Reply:
Thanks @sujeet-kr 
I'm removing "EKS with Calico CNI" from the Unsupported list.